src/Scripting/CSharpTest/ObjectFormatterTests.cs

```diff
@@ -418,7 +418,16 @@ public void DebuggerProxy_FrameworkTypes_IEnumerable()
 
             obj = Enumerable.Range(0, 10);
             str = s_formatter.FormatObject(obj, SingleLineOptions);
-            Assert.Equal("RangeIterator { 0, 1, 2, 3, 4, 5, 6, 7, 8, 9 }", str);
+
+            // the implementation differs between .NET Core and .NET FX
```

Comment:
Not a bug, just a different implementation. The object display is similar to a debugger display in immediate window, just implemented thru Reflection. It sometimes captures an implementation detail of the code. In one of the cases the method is implemented thru iterator (the object display actually decodes the compiler generated name of the iterator method), the other is implemented thru struct, I think.